compiler/ppx/ppx_optcomp_light.ml

```diff
@@ -285,7 +297,21 @@ let traverse =
             | Some pattern -> Some { case with pc_lhs = pattern })
       in
       super#cases cases
+
+    method! signature_item item =
```

Comment:
What about matching on signature instead of individual items ? Similar to what we do for structures?

Reply:
Your alternative is fine with me.